quantum/keyboard.c

```diff
@@ -171,9 +171,9 @@ static matrix_row_t get_real_keys(uint8_t row, matrix_row_t rowdata) {
     matrix_row_t out = 0;
     for (uint8_t col = 0; col < MATRIX_COLS; col++) {
         // read each key in the row data and check if the keymap defines it as a real key
-        if (keycode_at_keymap_location(0, row, col) && (rowdata & (1 << col))) {
+        if (keycode_at_keymap_location(0, row, col) && (rowdata & (((matrix_row_t)1) << col))) {
```

Comment:
Wouldn't `1UL << col` work here too?

Reply:
Yep, that should work too. I didn't know which style is preferred. Shall I change it?

Reply:
The code is fine, and guards against a future bumping of `matrix_row_t` to `uint64_t`, necessitating a `ULL` wart.